src/pattern.py

```diff
@@ -1,2 +1,13 @@
 
 # Print the pattern
+
+a = ["*"]
+b = ["*", "*"]
+c = ["*", "*", "*"]
+d = ["*", "*", "*", "*"]
+e = ["*", "*", "*", "*", "*"]
+
+pattern_list = [a, b, c, d, e, d, c, b, a]
+
+for lst in pattern_list:
+    print(*lst)
```

Comment:
Not exactly what I had in mind, but this certainly works. As long as the depth of the deepest line never changes :)